### DISCUSSION TITLE
Blazor Support Discussion

### DISCUSSION THREAD

**Discussion by KHANFAIZAN05** [dotnetcore/BootstrapBlazor#1285] [Q&A] [answered]
Hi I am Leading a Project Based On .net 7 . we finally decided to go with the BootstrapBlazor Ui Component Libray...

I want To Know Should  we use  it for Production Enterprise Application.. If Something in The Later Phase Went Broken Or Will Not work.. Should we Get The Community Support... Or May be There Can be A Subscription Support...

Please Let Us Know.....

Regards
Khan Faizan

**Comment by ArgoZhang** [comment]
@KHANFAIZAN05 

Glad you used our component library. 

Our team provides full support for various projects, especially bugs, which we will fix as soon as possible, generally within 2 days. For some functions that we do not have at present, we may not be able to provide them in a short time. For example, the table does not provide a `Grouping` function at present

**Reply by KHANFAIZAN05** [reply]
@ArgoZhang  it is good to hear that, and I quite Personally Like Your Library.  Thank You.  Keep Up the Good Work Man 👍👍

**Reply by ArgoZhang** [reply]
Thank you!

**Reply by KHANFAIZAN05** [reply]
@ArgoZhang  Can You Please Tell Me If it is possible for my team and me to Customize the Component(s) Styling, (Color Etc)

**Reply by densen2014** [reply]
like normally css, so easy to cover library's style

**Reply by KHANFAIZAN05** [reply]
@densen2014  normal css styling here and there

**Reply by ArgoZhang** [reply, marked as answer]
@KHANFAIZAN05 

### Example - customize style for `Select` Component

The styles of all components will be variable.

```
.select {
    --bb-select-focus-shadow: none;
    --bb-select-bg: #fff;
    --bb-select-padding: 6px 2.25rem 6px 0.75rem;
    --bb-select-append-width: 30px;
    --bb-select-append-color: #c0c4cc;
    --bb-select-search-padding: 7px 10px;
    --bb-select-search-margin-bottom: .5rem;
    --bb-select-search-border-color: var(--bs-border-color);
    --bb-select-search-padding-right: 30px;
    --bb-select-search-icon-color: var(--bb-select-search-border-color);
    --bb-select-search-icon-right: 18px;
    --bb-select-search-icon-top: 18px;
}
```

1. create a style file like `site.css` in your project and put it into `wwwroot` folder
2. add `<link xxx xxx>` html element on `_Layout.cshtml`
3. customer style in `site.css`
```
.select {
    --bb-select-bg: #f0f0f0;
}
```

finally. the style in your `site.css` will override the component style

### Note the style file order
```
    <link rel="stylesheet" href="~/_content/BootstrapBlazor/css/bootstrap.blazor.bundle.min.css" asp-append-version="true">
    <link rel="stylesheet" href="~/site.css" asp-append-version="true">
```

**Reply by KHANFAIZAN05** [reply]
@ArgoZhang  great..

So we can override the styles . And moreover is it bootstrap 5

**Reply by ArgoZhang** [reply]
sure! You can override any style you want.
